gomint-server/src/main/java/io/gomint/server/entity/EntityPlayer.java

```diff
@@ -150,6 +148,31 @@ public int getViewDistance() {
         return this.viewDistance;
     }
 
+    @Override
+    public void transfer( InetSocketAddress inetSocketAddress ) {
+        String address = inetSocketAddress.getAddress().getHostAddress();
+        int port = inetSocketAddress.getPort();
+        PacketTransfer packetTransfer = new PacketTransfer();
+        packetTransfer.address = address;
+        packetTransfer.port = port;
+        this.connection.send( packetTransfer );
+    }
+
+    @Override
+    public void setHealth( double amount ) {
+        if(amount < 1){
+            amount = 0;
+        }
+
+        AttributeInstance attributeInstance = this.attributes.get( Attribute.HEALTH.getKey() );
+        attributeInstance.setValue( (float) amount );
+        PacketUpdateAttributes updateAttributes = new PacketUpdateAttributes();
```

Comment:
There is no manual updating required. The EntityPlayer will update on next tick.